action.yaml

```diff
@@ -25,7 +21,7 @@ runs:
         # install pre-commit if needed and toggle how the action runs.
         echo "PRE_COMMIT_BIN=$(command -v pre-commit)" >> $GITHUB_ENV
     - name: Install Turo conventional commit configuration
-      if: inputs.no-commitlint != true
+      if: hashFiles(inputs.config-file) != ''
```

Comment:
Wondering if this could be done to eliminate some repeated text:

env:
  HAS_CONFIG_FILE: hashFiles(inputs.config-file) != ''

Then just use env.HAS_CONFIG_FILE below.